ui_testing/pages/testplan_page.py

```diff
@@ -240,3 +240,13 @@ def get_testunit_quantification_limit(self, testunits, testunit_display_name):
                 quantification_limit = testunit['Quantification Limit']
                 return quantification_limit
 
+    def edit_test_unit_quantification_limits(self, upper, lower):
```

Comment:
I'm not sure, but maybe there is already implemented functions to do this

Reply:
https://github.com/modeso-open-source/1lims-automation/blob/LIMS5300-4426/ui_testing/pages/testplan_page.py#L232
??

Reply:
I used it

Reply:
no no dont use the DB one

Reply:
sorry I mean this function 
https://github.com/modeso-open-source/1lims-automation/blob/LIMS5300-4426/ui_testing/pages/testplan_page.py#L236
def update_upper_lower_limits_of_testunit(self, old_upper, old_lower):
I used it and it worked well, thank you for your note